src/Widgets/DisplayWidget.vala

```diff
@@ -23,6 +23,10 @@ public class Power.Widgets.DisplayWidget : Gtk.Grid {
     private Gtk.Label percent_label;
     private bool allow_percent = false;
 
+    // The integer returned is a relative change in the brightness value (ie.
+    // a +10 increase or a -10 decrease)
+    public signal void brightness_change (int change);
```

Comment:
This signal is unneeded.  Make this Widget an EventBox (containing a grid) and let the scroll event be handled by Indicator.vala.